src/app/static/src/app/store/modelOutput/modelOutput.ts

```diff
@@ -9,87 +9,34 @@ export interface ModelOutputState {}
-            {
-                "indicator": "new_infections",
-                "value_column": "mean",
-                "indicator_column": "indicator_id",
-                "indicator_value": "7",
-                "name": "New Infections",
-                "error_low_column": "lower",
-                "error_high_column": "upper"
-            },
-            {
-                "indicator": "plhiv",
-                "value_column": "mean",
-                "indicator_column": "indicator_id",
-                "indicator_value": "3",
-                "name": "PLHIV",
-                "error_low_column": "lower",
-                "error_high_column": "upper"
-            },
-            {
-                "indicator": "population",
-                "value_column": "mean",
-                "indicator_column": "indicator_id",
-                "indicator_value": "1",
-                "name": "Population",
-                "error_low_column": "lower",
-                "error_high_column": "upper"
-            },
-            {
-                "indicator": "prevalence",
-                "value_column": "mean",
-                "indicator_column": "indicator_id",
-                "indicator_value": "2",
-                "name": "Prevalence",
-                "error_low_column": "lower",
-                "error_high_column": "upper"
-            }];
+        return rootState.modelRun.result!!.plottingMetadata.barchart.indicators;
     },
     barchartFilters: (state: ModelOutputState, getters: any, rootState: RootState): Filter[] => {
-        //TODO: Get these from ModelResultResponse
-        const filterOptions = rootState.modelRun.result!!.filters!!;
 
         const regions: FilterOption[] = rootState.baseline.shape!!.filters!!.regions ?
                                         [rootState.baseline.shape!!.filters!!.regions] : [];
 
-        return [
-            {
-                "id": "age",
-                "column_id": "age_group_id",
-                "label": "Age group",
-                "options": filterOptions.age || []
-            },
-            {
+        let filters = rootState.modelRun.result!!.plottingMetadata.barchart.filters;
+        //THIS IS A WORKAROUND FOR A BUG IN HINTR - take out when fixed!
+        //Column_id and label are the wrong way round for quarter
+        //(Also, sex is missing)
+        //Should get fixed as part of https://vimc.myjetbrains.com/youtrack/issue/mrc-577
+        let newFilters = null;
+        const quarter = filters.find((f: any) => f.id=="quarter");
+        if (quarter) {
+            const newQuarter = {
+                ...quarter,
+                column_id:  "quarter_id",
+                label: "Quarter"
+            };
+            newFilters = filters.filter((f: any) => f.id != "quarter");
+            newFilters.push(newQuarter);
+
+            filters = newFilters;
+        }
+        const sex = filters.find((f: any) => f.id == "sex");
+        if (!sex){
+            filters.push({
```

Comment:
So is `sex` sometimes not present, sometimes present? I know I was the one who originally suggested hard-coding filters we already knew about but now seems more clear that was the wrong call, can we just rely on it being returned by hintr?